index.js

```diff
@@ -0,0 +1,4 @@
+const prentElement = document.querySelector("ul");
+const childElement = document.createElement("li");
+childElement.innerText = "これです";
```

Comment:
Q
why did you use innerText instead of textContent

Reply:
Thanks for the review!
As you said, this time it was just simple text, so we will change it to textContent.